tools/benchmark_tool/openvino/tools/benchmark/utils/statistics_report.py

```diff
@@ -2,40 +2,57 @@
 # SPDX-License-Identifier: Apache-2.0
 
 from datetime import timedelta
+from io import TextIOWrapper
 import os
-import sys
 from enum import Enum
+import json
 import csv
+from typing import Dict, List, Tuple
 from .logging import logger
+import numpy as np
```

Comment:
do we need to add extra dependencies to requirements txt files?

Reply:
All of them are already present in the default Python (except for numpy, but it's already a requirement for benchmark app)